### ISSUE TITLE
File path suggestion doesn't work without inputting delimiters

### ISSUE THREAD

**Issue by falsandtru** [microsoft/TypeScript#26247]
**TypeScript Version:**  3.1.0-dev.20180804

**Search Terms:**

**Code**

```ts
import './'; // When I put a delimiter `/`, all files under the directory are suggested.
import './a'; // When I put a part of filename (not filtering), no files are suggested.
```

**Expected behavior:**

Suggestion works with filename inputting.

**Actual behavior:**

Suggestion works only with delimiter inputting.

**Playground Link:** 

**Related Issues:**

**Comment by andy-ms** [comment]
Should fix #26288 before investigating this.

**Comment by sheetalkamat** [comment]
I don't see the issue with the behavior. I have files: a.ts, aa.ts, b.ts, c.ts in the folder and I create d.ts and start typing contents from your repro
`
import './'; // I see a, aa, b, c
import './a'; // I see a, aa
import './x'; // I donot see anything (just like you don't get anything when getting completion at someVar.x if the type of someVar doesn't have properties starting with x)
`

**Comment by falsandtru** [comment]
Are you seeing a suggestion which is reopen by inputting `a`? Or filtered by `a`?

repro:

1. Clone and install https://github.com/falsandtru/spica.
1. Open `src/clock.tick.test.ts`.
1. Change `import { tick } from './clock';` to `import { tick } from './';`.
1. Close the suggestion if open.
1. Put `c` as a part of filename. (so the current input is `import { tick } from './c';`)
1. Should open the suggestion but it is actually not.

FYI:
os: win10
typescript: 3.1.0-dev.20180813
vscode: 1.26.0

**Comment by andy-ms** [comment]
Seems to be working on linux with `typescript@next`.

**Comment by falsandtru** [comment]
Indeed, I can see it. Can you repro on win10?

**Comment by falsandtru** [comment]
However, and anyway, file path suggestion doesn't work with many cases even on linux. This feature works only with very small cases.

**Comment by andy-ms** [comment]
Tested on windows and reproduced the issue. I don't see "completionInfo" being requested. CC @mjbvz

**Comment by falsandtru** [comment]
Thanks @andy-ms .

**Comment by andy-ms** [comment]
@mjbvz Could you take a look at this?

**Comment by mjbvz** [comment]
I can't repo this on win 10 wit the latest vscode insiders:

![screen shot 2018-09-10 at 11 51 10 am](https://user-images.githubusercontent.com/12821956/45317783-da73b680-b4ef-11e8-92dd-3dd63121afe3.png)

@falsandtru Does this still happen when you disable all extensions? Have you configured: `"typescript.quickSuggestionsForPaths"` in your user or workspace settings?

**Comment by falsandtru** [comment]
Indeed, the issue is fixed with vscode 1.27.1.

**Comment by falsandtru** [comment]
@RyanCavanaugh This issue lacks an important info that this issue is fixed once and reopened for regression. You should reopen any other issue opened for a regression rather than reopen this issue. Thus reopening this issue is bad on the issue management. **Your issue management is bad. You just reopened this issue to hide your failure and shame https://github.com/microsoft/TypeScript/issues/28523#issuecomment-511931860.**
